build.xml

```diff
@@ -13,6 +13,13 @@
 	<description>Collaborative Drug Discovery</description>
 
 	<property name="pkg" location="pkg" />
+	<property file="${basedir}/build.properties" />
```

Comment:
Regarding duplication of parameters:
```
Looks reasonable. Is there a way to avoid the duplication of the parameter setting for buildAPI and build?
```
I presume you are asking about why several calls to `antcall` declare the same parameters.  I personally like the "11" or "1.8" explicit in the parameter declarations; ditto for the `compilerarg` parameter values.

So, people inevitably will modify `build.properties` to suit their local environment.  Maybe what we can do is add it to the repo, but then ignore it in `.gitignore`.